app/indexers/publication_indexer.rb

```diff
@@ -1,6 +1,7 @@
 # frozen_string_literal: true
 class PublicationIndexer < BaseIndexer
   include IndexesEnglishLanguageDates
+  include IndexesCitationMetadata
```

Comment:
bibliographic_metadata is a part of our [CoreMetadata mixin](https://github.com/LafayetteCollegeLibraries/spot/blob/2022.5/app/models/concerns/spot/core_metadata.rb), applied to all work types, so maybe it should be a part of the BaseIndexer instead? thoughts?